e2e-tests/non-voting/compare/statefulset_nonvoting-rs0-nv.yml

```diff
@@ -0,0 +1,197 @@
+apiVersion: apps/v1
```

Comment:
we need to create compare files for OC too

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/723/commits/78b976cdc932c4381c8343c0811c45e8feddb755